common/luaobject.h

```diff
@@ -195,6 +195,16 @@ int luaA_object_emit_signal_simple(lua_State *);
         return 1; \
     }
 
+#define LUA_OBJECT_EXPORT_PROPERTY2(pfx, type, field, pusher, empty_value) \
```

Comment:
I squashed the following into this commit, is that ok? https://gist.github.com/psychon/d738c616db977eafff8149b1a1eff95f